ccd-definition/CaseEvent.json

```diff
@@ -349,10 +349,11 @@
     "PreConditionState(s)": "Submitted",
     "PostConditionState": "Submitted",
     "SecurityClassification": "Public",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/upload-c2/about-to-submit",
```

Comment:
just had a chat with you re: using one event for c2. Check out https://github.com/hmcts/fpl-ccd-configuration/pull/540/files as I believe it's a similar sort of thing implementation wise

@akomisarek, separate story to combine these? sounds like c2 can and will be uploaded anywhere from submitted onwards